### ISSUE TITLE
Verschönerung Hilfeseite mit CSS

### ISSUE THREAD

**Issue by innosmith** [decidim-swiss/decidim.swiss#60]
Prio 1: Ist es allenfalls möglich, bei den zwei letzten Themen Rechtliches die Texte ganz auszublenden, sodass nur noch analog den Hilfethemen oberhalb der Titel angezeigt wird?

Prio 2, falls Prio 1 nicht umsetzbar: Ist es möglich, auf der Hilfeseite von deinklima bei den zwei letzten Themen Impressum und rechtliche Hinweise einen Abstand zwischen Titel und Text einzufügen?

Prio 3: Ist es möglich, analog MeinQuartier siehe Screenshot auf der Hilfeseite Buttons für "Weiterlesen" einzublenden?

Wie gross wäre etwa der Aufwand für diese Anpassungen? Annahme: technisch lässt sich alles mit CSS anpassen.

![Google Chrome - Hilfe - deinklima ch 2023-05-06 um 12 29 29](https://user-images.githubusercontent.com/75796635/236618946-df2a3de4-31cb-4a00-881a-38ccfb2a0de8.png)
![Google Chrome - Informationen - Meinquartier Zuerich 2023-05-06 um 12 24 41](https://user-images.githubusercontent.com/75796635/236618950-a7f42faa-da89-4b7b-9e6c-e1fe436b6bab.png)

**Comment by carlobeltrame** [comment]
Gemäss Besprechung mit Anthony: Buttons via CSS wieder einblenden (ca. 0.5h), dann schauen wir weiter.

**Comment by innosmith** [comment]
Hoi @carlobeltrame : ich habe die Priorisierung der einzelnen Optimierungsmöglichkeiten gemäss Kundin in der Beschreibung des Tickets angepasst.

**Comment by Kagemaru** [comment]
Hallo @innosmith

Zu Prio 1:
Dies geht mit einer Code Änderung, ansonsten nicht. Da es auch hier zuwenig identifizierbare CSS Selektoren gibt.

Zu Prio 2:
Hier könnte ich falls es unabdingbar ist, eine Lösung mit CSS finden. Diese wäre aber Hacky, da es wieder zu wenige Identifikationsmerkmale gibt.

Zu Prio 3:
Bei den Hilfeseiten fehlt uns momentan eine Unterscheidung der card footer (auf denen sich die Buttons befinden) von denen auf anderen Seiten.
Daher ist eine kleine Code Anpassung nötig.

Wie du siehst laufen hier alle Änderungen auf eine Anpassung im Code hinaus. Soll ich es trotzdem noch angehen?

**Comment by innosmith** [comment]
Danke für die Abklärung @Kagemaru . Müsste bei einer Code-Änderung diese zusätzlich als Modul umgesetzt werden, da wir andere Tenants nicht beeinflussen wollen? Was denkst du bez. Umsetzungsaufwand?

**Comment by Kagemaru** [comment]
Nein, ich würde einfach zusätzliche Indikatoren zu der Hilfeseite hinzufügen, dann kann man den Rest mit CSS lösen.
Die anderen Tenants kriegen dann auch diese Indikatoren, sie ändern aber alleine nichts am Layout.

Code changes: ca. 1h
Prio1 (CSS): ca. 0.5h
Prio2 (CSS): entfällt
Prio3 (CSS): ca. 0.5h
Deployment ca. 0.5h
Total: 2.5-3h

**Comment by innosmith** [comment]
@Kagemaru : bitte umsetzen. Die Kundin möchte die Hilfeseite verschönern, sodass es schlussendlich ähnlich wie auf MeinQuartier aussieht https://meinquartier.zuerich/pages.

**Comment by Kagemaru** [comment]
@innosmith 
Ich habe es nun mal umgesetzt. Kannst du kurz testen?

**Comment by bsantschi** [comment]
Hoi @innosmith können wir die Umsetzung dieses Tickets heute Mittag auf PROD releasen?

**Comment by innosmith** [comment]
Sehr gerne @bsantschi .
